code/go/0chain.net/smartcontract/storagesc/config.go

```diff
@@ -45,14 +45,28 @@ type writePoolConfig struct {
 }
 
 type blockReward struct {
-	BlockReward             currency.Coin `json:"block_reward"`
-	BlockRewardChangePeriod int64         `json:"block_reward_change_period"`
-	BlockRewardChangeRatio  float64       `json:"block_reward_change_ratio"`
-	QualifyingStake         currency.Coin `json:"qualifying_stake"`
-	SharderWeight           float64       `json:"sharder_weight"`
-	MinerWeight             float64       `json:"miner_weight"`
-	BlobberWeight           float64       `json:"blobber_weight"`
-	TriggerPeriod           int64         `json:"trigger_period"`
+	BlockReward             currency.Coin    `json:"block_reward"`
+	BlockRewardChangePeriod int64            `json:"block_reward_change_period"`
+	BlockRewardChangeRatio  float64          `json:"block_reward_change_ratio"`
+	QualifyingStake         currency.Coin    `json:"qualifying_stake"`
+	SharderWeight           float64          `json:"sharder_weight"`
+	MinerWeight             float64          `json:"miner_weight"`
+	BlobberWeight           float64          `json:"blobber_weight"`
+	TriggerPeriod           int64            `json:"trigger_period"`
+	Gamma                   blockRewardGamma `json:"gamma"`
+	Zeta                    blockRewardZeta  `json:"zeta"`
+}
+
+type blockRewardGamma struct {
```

Comment:
add please  validation logic for new fields

Reply:
@sum-it08 can you check what validation should be applied for the Gama params

Reply:
Done!